credit_calculator.py

```diff
@@ -29,14 +29,16 @@ def months_until_paid_out(credit_sum, monthly_payment, initial_payment, monthly_
 if __name__ == "__main__":
     
     parser = argparse.ArgumentParser(description='Arguments required to do the calculations')
-    parser.add_argument('credit_sum', type=float, help='Credit Summary')
-    parser.add_argument('monthly_payment', type=float, help='Monthly Payment')
+    parser.add_argument('credit_sum', type=float, help='--sum')
```

Comment:
@Ghaust 
It should look more like this: `parser.add_argument('--sum', type=float, help='Credit sum')`

This way you can call script `python credit_calculator.py --sum 3000`